unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/WorldChatWindowController.cs

```diff
@@ -380,7 +382,17 @@ private void HandleChannelUpdated(Channel channel)
         
         isRequestingChannels = false;
     }
-    
+
+    private void HandleChannelJoined(Channel channel)
+    {
+        OpenPublicChannel(channel.ChannelId);
+    }
+
+    private void HandleJoinChannelError(string channelId, string message)
+    {
+        // TO DO: Give error feedback...
```

Comment:
you could implement an event

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2568/commits/f618515e2d7696d0e12303aeb3e0c8acd9fb8c99.